desktop/release-notes.md

```diff
@@ -24,13 +24,63 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.2
+
+{% include release-date.html date="2023-01-19" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed image tagging with an existing tag with the containerd integration.
```

Comment:
@rumpl I don’t quite understand what this means. Does it have to do with overwriting a tag when building an image, or something like that?

Reply:
It's fixing `docker tag old new` when the `new` image already exists, the side-effect is that `docker buildx build` doesn't end in an error any more

Reply:
Any ideas how to say it in a nicer way?

Reply:
```suggestion
- Fixed `docker tag` not overwriting the target tag if it already exists (with containerd integration feature enabled)
```
Maybe something like this?

Reply:
How's this?

```suggestion
- Fixed an issue where `docker tag` would overwrite existing tags when the containerd image store was enabled.
```

Reply:
"would not" / "would produce an error".

The actual failure was that `docker build` would always fail with en error `image "<IMAGE NAME>": already exists`


```bash
docker build -t myimage .
...
image "docker.io/library/myimage:latest": already exists


echo $?
1
```

Reply:
So
```suggestion
- Fixed an issue where `docker tag` would not overwrite existing tags when the containerd image store was enabled.
```
?

Reply:
Or perhaps;

> Fixed an issue where `docker build` and `docker tag` commands produced an `image already exists` error if the containerd integration feature was enabled.


@dvdksn @rumpl any preference / suggestions?

Reply:
Let's go with your suggestion @thaJeztah

Reply:
(just formatting it as a suggested change to be able to click on the commit button)
```suggestion
- Fixed an issue where `docker build` and `docker tag` commands produced an `image already exists` error if the containerd integration feature is enabled.
```

Reply:
@aevesdocker does 👆 work for you?

Reply:
Yes, LGTM!